src/CoreWCF.WebHttp/src/CoreWCF/Dispatcher/QueryStringConverter.cs

```diff
@@ -0,0 +1,331 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.Globalization;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using System.Xml;
+using CoreWCF.Runtime;
+
+namespace CoreWCF.Dispatcher
+{
+    public class QueryStringConverter
+    {
+        readonly Hashtable _defaultSupportedQueryStringTypes;
+
```

Comment:
Missing accessibility modifier in multiple places

Reply:
Addressed in a39c44fda0baf05c7b936cf0987bd12657e6dc14.